apps/wallet/src/modals/FirstTimePopup/PixeverseCard.tsx

```diff
@@ -0,0 +1,91 @@
+import { Image, StyleSheet, View } from 'react-native';
+import { Text } from '@walless/gui';
+import { mockWidgets } from 'state/widget';
+
+const PixeverseCard = () => {
+	const pixeverseWidget = mockWidgets.find((item) => item._id === 'pixeverse');
+	if (!pixeverseWidget) return null;
```

Comment:
this condition should be check outside this component, assume that this one is only work with available pixeverse widget data via prop